sys-apps/yarn/yarn-0.17.9.ebuild

```diff
@@ -0,0 +1,37 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+DESCRIPTION="Fast, reliable, and secure node dependency management."
+HOMEPAGE="https://yarnpkg.com"
+SRC_URI="https://github.com/yarnpkg/yarn/releases/download/v${PV}/yarn-v${PV}.tar.gz"
+
+LICENSE="BSD-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="+system-node"
+
+RDEPEND="system-node? ( net-libs/nodejs )"
+
+DEPEND="${RDEPEND}"
+
+S="${WORKDIR}/dist"
+
+src_install() {
+	default
```

Comment:
What does the default do here? Because it seems to me that you are doing all the install below ;-).